### PR TITLE
add initial FAQ for BQ OAuth related to Google Drive scope

### PR DESCRIPTION
**Pull request by coapacetic**
## What are you changing in this pull request and why?
The BigQuery OAuth application requires some scopes to access Google Drive files. This is adding a quick FAQ to the bottom of the page in order to clarify why this is needed as part of the BQ OAuth connection and links to external docs from the Google team.

## Checklist
- [ ] Review the [Content style guide](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/content-style-guide.md) and [About versioning](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#adding-a-new-version) so my content adheres to these guidelines.
- [ ] Needs a technical review from the adapter team as a second opintion

### CODE REVIEW

File: website/docs/faqs/Warehouse/bq-oauth-drive-scope.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+---
+title: Why does the BigQuery OAuth application require scopes to Google Drive
+description: "Use this quick explainer to learn more about scopes in the BigQuery OAuth applicaton"
```
Comment (runleonarun) [inline website/docs/faqs/Warehouse/bq-oauth-drive-scope.md]:
```suggestion
description: "Learn more about Google Drive scopes in the BigQuery OAuth application"
```

File: website/docs/faqs/Warehouse/bq-oauth-drive-scope.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+sidebar_label: "BigQuery OAuth GDrive Scopes"
+id: bq-oauth-drive-scope
+---
+
+BigQuery supports external tables over both personal Drive files and shared files. For more information on Drive, see Drive training and help.
```
Comment (runleonarun) [inline website/docs/faqs/Warehouse/bq-oauth-drive-scope.md]:
```suggestion
BigQuery supports external tables over both personal Google Drive files and shared files. For more information, refer to [Create Google Drive external tables](https://cloud.google.com/bigquery/docs/external-data-drive).
```

File: website/docs/faqs/Warehouse/bq-oauth-drive-scope.md [hunk 3]
```diff
@@ -0,0 +1,10 @@
+---
+title: Why does the BigQuery OAuth application require scopes to Google Drive
```
Comment (runleonarun) [inline website/docs/faqs/Warehouse/bq-oauth-drive-scope.md]:
```suggestion
title: Why does the BigQuery OAuth application require scopes to Google Drive?
```

File: website/docs/faqs/Warehouse/bq-oauth-drive-scope.md [hunk 4]
```diff
@@ -0,0 +1,10 @@
...
+description: "Use this quick explainer to learn more about scopes in the BigQuery OAuth applicaton"
+sidebar_label: "BigQuery OAuth GDrive Scopes"
```
Comment (runleonarun) [inline website/docs/faqs/Warehouse/bq-oauth-drive-scope.md]:
```suggestion
sidebar_label: "BigQuery OAuth Drive Scopes"
```

File: website/docs/faqs/Warehouse/bq-oauth-drive-scope.md [hunk 5]
```diff
@@ -0,0 +1,10 @@
...
+id: bq-oauth-drive-scope
+---
+
+BigQuery supports external tables over both personal Drive files and shared files. For more information on Drive, see Drive training and help.
+
+You can read more about this in the [BigQuery docs](https://cloud.google.com/bigquery/docs/external-data-drive) on this particular topics.
```
Comment (runleonarun) [inline website/docs/faqs/Warehouse/bq-oauth-drive-scope.md]:
```suggestion
```

**Review by runleonarun — COMMENTED** [review]
Hey @coapacetic! Thanks for opening this PR!! I have some suggestions ~~and we need to add the new FAQ to the TOC~~. (oops sidebar is auto-generated) If these suggestions look OK, one of us can do the rest and publish it.

**Review by mirnawong1 — APPROVED** [review]
lgtm @runleonarun @coapacetic ! thank you and have pushed all the changes in, however, let me know if there are any objections ✨

**Comment by coapacetic** [comment]
Just catching up now and seeing that it was merged -- happy to contribute
and thank you Mirna for picking it up and running with it!

Kyle

**Pull request merged**